### PR TITLE
feat (wave): add Wave adapter to docs

### PR DESCRIPTION
**Pull request by jason-gill00**
Add Wave to https://developer.tryfinch.com/docs/reference/96f5be9e0ec1a-providers

### CODE REVIEW

**Comment by alexromano** [comment]
we should add compatibility sections as well yeah?

**Comment by jason-gill00** [comment]
> we should add compatibility sections as well yeah?

I think @seii updated the compatibility screenshots. 

https://developer.tryfinch.com/docs/reference/0517ab806dda4-compatibility

**Comment by seii** [comment]
@jason-gill00 @alexromano Yes, I had updated the Compatibility page images from the spreadsheet. I didn't realize at the time that Wave wasn't listed in the Providers page, I'll keep an eye on that next time.

**Comment by alexromano** [comment]
this integration isn't launched yet. why is it shown in the public docs?

**Comment by seii** [comment]
@alexromano Because it was in the Compatibility spreadsheet, and that spreadsheet has never previously contained sensitive information to my knowledge. If we are going to populate that information prior to the public release of an adapter, then I can ask Engineering before updating those images in the future.

**Review by sshariff01 — APPROVED** [review]

**Review by sshariff01 — CHANGES_REQUESTED** [review]
Actually https://github.com/Finch-API/finch-docs/blob/main/docs/Development-Guides/Providers.md?plain=1#L188 needs to be removed too.

File: docs/Development-Guides/Providers.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Sequoia One | `sequoia_one` | <span style="color:darkgreen">Automated</span> | <
 Square Payroll | `square_payroll` | <span style="color:darkgreen">Automated</span> | <span style="color:goldenrod">Assisted</span>
 TriNet | `trinet` | <span style="color:darkgreen">Automated</span> | <span style="color:goldenrod">Assisted</span>
 UltiPro (UKG Pro) | `ulti_pro` | <span style="color:darkgreen">Automated</span> | <span style="color:goldenrod">Assisted</span>
+Wave | `wave` | <span style="color:darkgreen">Automated</span> | <span style="color:goldenrod">Assisted</span>
```
Comment (sshariff01) [inline docs/Development-Guides/Providers.md:35]:
```suggestion
Wave Payroll | `wave` | <span style="color:darkgreen">Automated</span> | <span style="color:goldenrod">Assisted</span>
```

**Review by sshariff01 — APPROVED** [review]

**Pull request merged**